examples/fluid/dam_break_2phase_2d.jl

```diff
@@ -63,21 +67,18 @@ end
 
 air_system = union(air_system, air_system2)
 
-# We use background_pressure here to prevent negative pressure regions in the gas phase.
 air_system_system = WeaklyCompressibleSPHSystem(air_system, fluid_density_calculator,
                                                 StateEquationCole(; sound_speed,
                                                                   reference_density=air_density,
                                                                   exponent=1,
                                                                   clip_negative_pressure=false,
-                                                                  background_pressure=1000),
+                                                                  background_pressure=0),
```

Comment:
Is it ever useful to overwrite the background pressure here? This is unphysical if we don't do the same for the fluid, no?